cmd/cspc-operator/app/storagepool_create.go

```diff
@@ -174,7 +179,7 @@ func (pc *PoolConfig) GetPoolDeploySpec(cspi *apis.CStorPoolInstance) (*appsv1.D
 					volume.NewBuilder().
 						WithName("tmp").
 						WithHostPathAndType(
-							getSparseDirPath()+"/shared-"+pc.AlgorithmConfig.CSPC.Name,
+							env.GetOpenebsBaseDirPath()+"/cstor-pool/"+pc.AlgorithmConfig.CSPC.Name,
```

Comment:
migration from SPC to CSPC need to be taken care due to this cc: @shubham14bajpai

Reply:
Sure @vishnuitta. Just one small question, are we going to ask users to come to the latest version of SPC before migration? If yes then upgrade will take care of this automatically.

Reply:
yes.. we should do migration from latest openebs version